src/Configurator/EnvConfigurator.php

```diff
@@ -80,6 +80,7 @@ private function configurePhpUnit(Recipe $recipe, $vars)
                     $data .= '        <!-- '.$value." -->\n";
                 } else {
                     $value = $this->options->expandTargetDir($value);
+                    $value = trim($value, '"');
```

Comment:
What would I do now if I wanted to pass a string that was enclosed by double quotes?

Reply:
I agree, according to the xml specs it would be better to use `htmlspecialchars` 

> To allow attribute values to contain both single and double quotes, the apostrophe or single-quote character (') may be represented as "&apos;", and the double-quote character (") as "&quot;".
https://www.w3.org/TR/xml11/#syntax

Just pushed the fix